.github/workflows/platformDeploy.yml

```diff
@@ -235,6 +253,24 @@ jobs:
         env:
           VERSION: ${{ env.IOS_VERSION }}
 
+      - name: Warn deployers if iOS production deploy failed
+        if: ${{ failure() && fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
+        uses: 8398a7/action-slack@v3
+        with:
+          status: custom
+          custom_payload: |
+            {
+              channel: '#deployer',
+              attachments: [{
+                color: "#DB4545",
+                pretext: `<!subteam^S4TJJ3PSL>`,
+                text: `💥 iOS production deploy failed. Please manually submit ${{ env.IOS_VERSION }} in the App Store. 💥`,
```

Comment:
Can we add links to `App Store`? https://appstoreconnect.apple.com/apps/1530278510/appstore would be nice!

Reply:
https://play.google.com/console/u/0/developers/8765590895836334604/app/4973041797096886180/releases/overview is android